obspy/taup/__init__.py

```diff
@@ -382,8 +438,9 @@
 
 
 # Convenience imports.
-from .tau import TauPyModel  # NOQA
-
+from .tau import TauPyModel # NOQA
+from .tau import plot_travel_times # NOQA
+from .tau import plot_ray_paths # NOQA
```

Comment:
not sure why flake8 is not complaining, but according to PEP8, there should be two spaces before the inline comment hash sign (compare the deleted line above) (https://www.python.org/dev/peps/pep-0008/#inline-comments)